MIP84/MIP84.md

```diff
@@ -80,61 +88,128 @@ This MIP will activate the Protocol-Owned Vault Emulation (POVE). It is an appro
 
 The POVE will operate based on manual processes defined in this MIP that are controlled directly by MKR holders using Executive Votes. The Risk Core Unit is mandated with following these pre-defined processes, making parameter adjustment proposals, and is given the emergency powers to de-risk POVE activity if it becomes necessary, including the immediate cessation of additional collateral purchases. This can be done through weekly governance polls, or if necessary be bundled directly into weekly or emergency Executive Votes.
 
-The POVE is exclusively controlled and owned by Maker Governance, and no third-party, individual, multisig or other point of control has any technical access to the POVE or its assets in any way.
+The actions that can be proposed by Risk Facilitators to derisk the POVE in a weekly governance poll, or directly into an emergency or weekly executive vote include, but are not limited to:
```

Comment:
I'd rephrase this or break it down into independent clauses, like:
- "RISK-001 Facilitators can propose actions to derisk the POVE via weekly governance polls or into emergency or weekly executive votes. These actions include but are not limited to:"